Makefile

```diff
@@ -4,8 +4,14 @@ help:
 	@echo "--- Commands for local development ---"
 	@echo "insights-upload-data       upload data to local ingress service for ROS processing"
 
+ros-upload-data:
+	curl -vvvv -F "upload=@sample-files/ros-sample.tar.gz;type=application/vnd.redhat.resource-optimization.sample+tgz" -F 'metadata={"branch_info": {"remote_branch": -1, "remote_leaf": -1}, "bios_uuid": "25d6ad97-60fa-4d3e-b2cc-4aa437c28f71", "ip_addresses": ["10.74.255.52", "2620:52:0:4af8:21a:4aff:fe00:a8a"], "fqdn": "vm255-52.gsslab.pnq2.redhat.com", "mac_addresses": ["00:1a:4a:00:0a:8a", "00:00:00:00:00:00"], "satellite_id": -1, "subscription_manager_id": "7846d4fa-6fcc-4b84-aa13-5f12e588ecca", "insights_id": "5ae6c008-b6b3-459b-82a0-7ebf75a047fd", "machine_id": "25d6ad97-60fa-4d3e-b2cc-4aa437c28f71"}' \
```

Comment:
IIUC, temporarily you have passed `metadata` details while request as tar is not having those details. 
will it be good to extract this metadata value in separate variable or json file? I found bit difficulty in reading this request. 
what do you think?